docs/concepts/overview/kubernetes-api.md

```diff
@@ -77,7 +77,7 @@ The API group is specified in a REST path and in the `apiVersion` field of a ser
 Currently there are several API groups in use:
 
 1. the "core" (oftentimes called "legacy", due to not having explicit group name) group, which is at
-   REST path `/api/v1` and is not specified as part of the `apiVersion` field, e.g. `apiVersion: v1`.
+   REST path `/api/v1` and is specified as part of the `apiVersion` field, e.g. `apiVersion: v1`.
```

Comment:
it is meant that the `apiVersion` value **is not** `/v1` (i.e. empty group name), but `v1`.

Reply:
@ahmetb @sttts thanks for clarification. it would be more clear to newcomers if we remove the use of **not** here and just suggest using `apiVersion: v1`.

Reply:
maybe we should reformulate the sentence and avoid "be or not be specified". Seems to be hard to understand.

Reply:
So, should the sentence be:

The *core* group, often referred to as *legacy*, is at REST path `/api/v1` and can be accessed using `apiVersion: v1`.

Reply:
"can be accessed using ..." still sounds odd.

How about

```
The core group, often referred to as the "legacy group", is at the REST path /api/v1 and uses `apiVersion: v1`.
```

E.g. the same wording as in the next paragraph.